src/common.h

```diff
@@ -45,7 +64,7 @@ struct ParseTree {
     wUPDATE,
     DELETE
   } commandType;
-  char* table;
+  char* table; // name of table
   struct Field* fields;
   struct Where* whereConstraints;
   struct Field* updateFields;
```

Comment:
I was thinking about this here and I'm kinda going back and forth on whether we need `updateFields` AND `insertFields` members here. We could leave it generic and let the context (aka `commandType`) decide. If `commandType` is `INSERT`, then fields are to be inserted, `UPDATE`, then update etc. etc.

But not willing to change if it breaks anybody's code though!